cmd/cluster-agent/subcommands/metamap/command.go

```diff
@@ -48,10 +48,10 @@ as well as which services are serving the pods. Or the deployment name for the p
 				fx.Supply(cliParams),
 				fx.Supply(core.BundleParams{
 					ConfigParams: config.NewClusterAgentParams(globalParams.ConfFilePath),
-					SecretParams: secrets.NewEnabledParams(),
 					LogParams:    log.ForOneShot(command.LoggerName, command.DefaultLogLevel, true),
 				}),
 				core.Bundle(),
+				secretsfx.Module(),
```

Comment:
I'm pretty sure we don't need secrets for this command.
`metamap` just makes a query to the DCA to show a mapping between pods and the kubernetes services they belong to.